src/test/groovy/com/jaredsburrows/license/LicensePluginAndroidSpec.groovy

```diff
@@ -1212,4 +1212,171 @@ final class LicensePluginAndroidSpec extends Specification {
     taskName << ['licenseDebugReport', 'licenseReleaseReport']
     copyEnabled << [true, false]
   }
+
```

Comment:
@jaredsburrows Do you want me to add these tests to the LicensePluginJavaSpec too?